docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -217,3 +217,49 @@ function hs_paragraph_types_update_10004(&$sandbox) {
   }
   $sandbox['#finished'] = count($sandbox['ids']) ? 1 - count($sandbox['ids']) / $sandbox['total'] : 1;
 }
+
+/**
+ * Set default field_accordion_style value for existing content.
+ */
+function hs_paragraph_types_update_10005(&$sandbox) {
```

Comment:
Why is the update necessary? In the preprocess you are checking for `secondary`. Seems to me that `default` and `null` will behave the same.

Reply:
@pookmish We just want to make things easier for users. If an user edits a page with accordions, they'll have to manually set the field in all existing accordions to "Default" before saving, because it's a required field.

Reply:
ok, yeah, see my comment just before yours.

Reply:
@pookmish Thanks! Yeah, it makes sense to remove the option. I updated the config and removed the update hook. 

The only detail is that we need to keep the "Default" label for usability purposes. I added a `hook_field_widget_complete_WIDGET_TYPE_form_alter()` to change the default "- None -" label to "Default". Please take a look and let me know if that works.

Reply:
Yeah that looks good. less impact on the code deployment as well. I wish there was a UI option to change the `_none` label.
The only possible thing that you could do is use `$context` [in this part](https://github.com/SU-HSDO/suhumsci/pull/1507/files#diff-c0831e03dcd9e444fe998d19c7d63430991498d1eba660f25f1ad6a6116196b5R147) in the hook to determine if it's the expected field. But it works the same way.